src/server.rs

```diff
@@ -1049,6 +1049,21 @@ impl<B: Buf> SendResponse<B> {
             .map_err(Into::into)
     }
 
+    /// Send a 100-continue response to a client request.
+    ///
+    /// The [`SendResponse`] instance is already associated with a received
+    /// request.  This function may only be called once per instance and only if
+    /// [`send_reset`] or [`send_response`] has not been previously called.
+    ///
+    /// [`SendResponse`]: #
+    /// [`send_reset`]: #method.send_reset
+    /// [`send_response`]: #method.send_response
+    pub fn send_continue(&mut self) -> Result<(), crate::Error> {
```

Comment:
It might be out-of-scope to suggest this, but should we perhaps consider a slightly broader method, that allows sending other informational responses, like `103 Early Hints` or future codes? In that case, we'd want to be able to send headers too...

Reply:
I thought about it when writing the PR, but avoided it since I'm not sure what it's signature should be.  `send_info(Response<()>)` feels a bit too wide, given that practically only 1xx responses are allowed here. OTOH, `send_response(Response<()>)` is _also_ wider than it technically supports. (Extensions are cleared, the body unused, and the only `Version` supported really, is HTTP2), so maybe such API is fine.